app/Models/Country.php

```diff
@@ -10,4 +10,24 @@ class Country extends Model
     use HasFactory;
 
     protected $fillable = ['name', 'short_code'];
+
+    public function medals()
```

Comment:
@kjarya I would call this method `sports()` - then you don't need to define those separate columns